posttest.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var findTables = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'T\'' + ';';
+var findViews = 'SELECT TABNAME FROM SYSCAT.TABLES WHERE TABSCHEMA = ' +
+  '\'' + schemaName + '\'' + ' and type=\'V\'' + ';';
+
+if (!process.env.CI)
+  return console.log('Skipping post-test ...');
+console.log('> Cleaning up the database ...');
+
+// drop all the tables and views from the schema
```

Comment:
wait....sorry if you are sure yours work fine you don't have to change :( My comment is more like I could not figure out how something works, not I am sure something doesn't work :-p